example.json

```diff
@@ -20,8 +20,16 @@
     "_showScaleIndicator": true,
     "_showScale": true,
     "_showScaleNumbers": true,
-    "labelStart": "incomplete",
-    "labelEnd": "complete",
+    "_labels": {
+        "_start": {
+            "text": "Start label",
+            "ariaLabel": "Start aria label"
```

Comment:
as people tend to copy-paste from example.json I think it might be better to have the default value in here be something more readily usable... maybe 'Start of the scale'?